include/groonga/groonga.h

```diff
@@ -906,6 +906,63 @@ GRN_API grn_obj *
 grn_ctx_db(grn_ctx *ctx);
 GRN_API grn_obj *
 grn_ctx_get(grn_ctx *ctx, const char *name, int name_size);
+/**
+ * \brief Retrieve all tables from the database the database that the context is
+ *        using.
```

Comment:
```suggestion
 * \brief Retrieve all tables from the database that the context is using.
```

`the database` は1個で十分？

Reply:
fix: [b9d66f2](https://github.com/groonga/groonga/pull/2129/commits/b9d66f20a001cff607443631785f7c4cee17dbd2) 十分でしたmm